docs/features/features-walk-through/access_requests.md

```diff
@@ -0,0 +1,36 @@
+# Request access to restricted files of a record
+
+You can allow authenticated and non-authenticated (guest) users to request access to view the restricted files of a public record. Access can be set to expire on a specific date as well as never expiring.
+
+This can be useful for record owners to manage access to restricted files of each record. For not authorized users, it gives the possibility to request access to the files.
```

Comment:
```suggestion
You can allow authenticated and non-authenticated (guest) users to request access to view the restricted files of a public record. Access can be set to expire on a specific date as well as never expire.

This can be useful for record owners to manage access to restricted files of each record. For unauthorized users, it gives the possibility to request access to the files.
```

Reply:
Thanks a lot! Done ✅